taproot.html

```diff
@@ -6,8 +6,10 @@
 <title>Bitcoin Taproot Activation</title>
 <style type="text/css">
 body {
+	margin: 0;
 	padding-top: 3em;
 	text-align: justify;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
Prefer to leave it as the user's preference (ie, default)

Reply:
This will leave the font styling likely untouched for the majority of the users.
This code will use the operating system's preferred font, using the so called "System font stack" (although I missed out on Linux distros in the current iteration here).

Reply:
If the user has not configured their default font, it will (unset) be the browser or OS's preference.